src/mockDebug.ts

```diff
@@ -16,7 +16,7 @@ import path = require('path');
 const VARIABLE_SEPARATOR = '%';
 
 function isComplexType(type: string) {
-	return ['Class', 'dw.', 'dw/', 'Object', 'Array'].some(ctype => type.includes(ctype));
+	return ['Class', 'dw.', 'dw/', 'object', 'Object', 'Array'].some(ctype => type.includes(ctype));
```

Comment:
this may cause a false positive for `null` ('null' is an object in JS)

Reply:
Thank you for the feedback.  

Currently the debugger is checking if the value property of each member/variable is null inside of each of the calls to the isComplexType method but after the variable is added to the _variableHandles using the create() method. 

The values are being mapped as null but are being added to the variables tree without issue:
![image](https://user-images.githubusercontent.com/16811151/36996266-9480c9d0-2073-11e8-9486-4179b44de39c.png)

It seems to me that there are a couple of ways that I could address this issue to ensure that the value is not null before the variable handle is created:

1. Change the isComplexType method to take the entire member/variable as an argument instead of the just the type property, and then verify that member.value !== null at the same time.
2. Do a separate check in the same conditionals where the isComplexType method is called to ensure that the value is not null:  
```javascript
    if (isComplexType(member.type) && member.value !== null) {}
```

Do you have a preference for how this should be addressed?

Reply:
@SqrTT  -- PR updated by adding a null check for the value property each time.